src/Geopilot.Frontend/vite.config.js

```diff
@@ -28,7 +28,57 @@ const keyFilePath = path.join(baseFolder, `${certificateName}.key`);
 // https://vitejs.dev/config/
 // noinspection JSUnusedGlobalSymbols
 export default defineConfig({
-  plugins: [react(), viteTsconfigPaths()],
+  plugins: [
+    react(),
+    viteTsconfigPaths(),
+    // Simple middleware to serve markdown files from src/assets/docs
+    {
+      name: "devPublic",
+      apply: "serve",
+      configureServer(server) {
+        server.middlewares.use((req, res, next) => {
+          if (!req?.url) {
+            return next();
+          }
+
+          // Get clean path without query params
+          const urlPath = req.url.split("?")[0];
+          const relativePath = urlPath.startsWith("/") ? urlPath.substring(1) : urlPath;
+          const filePath = path.resolve(process.cwd(), "devPublic", relativePath);
+
+          try {
+            if (fs.existsSync(filePath) && fs.statSync(filePath).isFile()) {
+              // Basic content type mapping for common file types
+              const ext = path.extname(filePath).toLowerCase();
+              const contentTypes = {
+                ".html": "text/html",
+                ".css": "text/css",
+                ".js": "application/javascript",
+                ".json": "application/json",
+                ".png": "image/png",
+                ".jpg": "image/jpeg",
+                ".jpeg": "image/jpeg",
+                ".gif": "image/gif",
+                ".svg": "image/svg+xml",
+                ".md": "text/markdown",
+                ".txt": "text/plain",
+                ".pdf": "application/pdf",
```

Comment:
Strong PP: Nur types unterstützten, welche wir aktuell benutzen.
```suggestion
                ".md": "text/markdown",
```

Reply:
Q: Wie soll `devPublic `zukünftig verwendet werden? Meine bisherige Annahme war, dass `devPublic `wie das normale `public` funktionieren soll, ohne das wir letzteres verwenden müssen (Grund dafür ist ja, dass der Public-Ordner für unsere Deployments ein bisschen kritisch ist, und Veränderungen zu unerwarteten Fehlern führen können). D.h. es sollten auch alle Dateien darin "geserved" werden.

Wenn das eine falsche Annahme ist, und dieser Ordner nur Markdowns beherbegen für unsere Footer soll, sollten wir ihn umbenennen.

Ich habe den Code in der neuen Version angepasst. Bitte gib mir hier Feedback.